lib/model/migrations/20210423-01-add-name-to-form-def.js

```diff
@@ -19,17 +19,12 @@ const up = async (db) => {
   });
 
   await db.raw('ALTER TABLE form_defs DISABLE TRIGGER check_managed_key');
-
-  const work = [];
-  for await (const def of db.select('*').from('form_defs').stream()) {
-    const partial = await Form.fromXml(def.xml);
-    if (partial.def.name == null) continue;
-
-    const data = { name: partial.def.name };
-    work.push(db.update(data).into('form_defs').where({ id: def.id }));
-  }
-  await Promise.all(work);
-
+  await db.raw(`update form_defs
+set name = (xpath(
+  '/*[local-name() = ''html'']/*[local-name() = ''head'']/*[local-name() = ''title'']/text()',
+  xml::xml
+))[1]::text
```

Comment:
`xpath()` returns an array of `xml` values. For an ODK XForm, the resulting array should either have one element or (if the form has no title) be empty. `[1]` will return `null` if the array is empty.